packages/account/src/Sections/Verification/ProofOfIncome/proof-of-income-form.tsx

```diff
@@ -0,0 +1,227 @@
+import React from 'react';
+import { Field, Formik, Form, FormikErrors, FormikHelpers, FormikValues } from 'formik';
+import { AccountStatusResponse, DocumentUploadRequest } from '@deriv/api-types';
+import {
+    Autocomplete,
+    Button,
+    DesktopWrapper,
+    FormSubmitErrorMessage,
+    MobileWrapper,
+    SelectNative,
+} from '@deriv/components';
+import { useFileUploader } from '@deriv/hooks';
+import { localize, Localize } from '@deriv/translations';
```

Comment:
@yauheni-deriv 
could we replace `localize` usage with `Localize` in this file?

Reply:
@heorhi-deriv, no buddy. The components require strings here...